### PR TITLE
Fix all uses of IN filter

### PR DESCRIPTION
**Pull request by nickpell**
@snopoke @kaapstorm @czue @lwyszomi 

Replaces https://github.com/dimagi/commcare-hq/pull/8852

fixes http://manage.dimagi.com/default.asp?185918

### CODE REVIEW

File: custom/world_vision/sqldata/child_sqldata.py [hunk 1]
```diff
@@ -186,7 +188,7 @@ def columns(self):
         return [
             DatabaseColumn("Total births",
                            CountUniqueColumn('doc_id',
-                                             filters=[AND([IN('mother_id', 'mother_ids'),
+                                             filters=[AND([IN('mother_id', get_INFilter_bindparams('mother_id', self.config['mother_ids'])),
```
Comment (dimagimon) [inline custom/world_vision/sqldata/child_sqldata.py]: E501 line too long (140 > 115 characters)

File: custom/world_vision/sqldata/child_sqldata.py [hunk 2]
```diff
@@ -374,7 +381,7 @@ def columns(self):
                                              filters=self.filters + [NOTEQ('weight_birth', 'empty')])),
             DatabaseColumn("Total births",
                            CountUniqueColumn('doc_id',
-                                             filters=[AND([IN('mother_id', 'mother_ids'),
+                                             filters=[AND([IN('mother_id', get_INFilter_bindparams('mother_id', self.config['mother_ids'])),
```
Comment (dimagimon) [inline custom/world_vision/sqldata/child_sqldata.py]: E501 line too long (140 > 115 characters)

File: custom/world_vision/sqldata/child_sqldata.py (hunk 1 above)
Comment (dimagimon) [inline custom/world_vision/sqldata/child_sqldata.py]: E501 line too long (140 > 115 characters)

File: custom/world_vision/sqldata/child_sqldata.py (hunk 2 above)
Comment (dimagimon) [inline custom/world_vision/sqldata/child_sqldata.py]: E501 line too long (140 > 115 characters)

**Comment by nickpell** [comment]
:white_check_mark: @calellowitz

File: custom/intrahealth/sqldata.py [hunk 3]
```diff
@@ -78,6 +85,10 @@ def calculate_total_row(self, rows):
                     total_row.append('')
         return total_row
 
+    @property
+    def filter_values(self):
+        return clean_IN_filter_value(super(BaseSqlData, self), 'archived_locations')
```
Comment (nickpell) [inline custom/intrahealth/sqldata.py]: forgot to call the baseclass property, made a fix for it, then accidentally pushed a62aa08a810e5cd90f2a40904ab06ebf659c2690 to master :fearful: So ... fixed now.

**Comment by kaapstorm** [comment]
:+1:

**Pull request merged**